### DISCUSSION TITLE
v0.3.3

### DISCUSSION THREAD

**Discussion by dworthen** [microsoft/graphrag#1117] [Announcements]
## What's Changed
- patch: Add entrypoints for incremental indexing
- patch: Clean up and organize run index code
- patch: Consistent config loading. Resolves #99 and Resolves #1049
- patch: Fix circular dependency when running prompt tune api directly
- patch: Fix default settings for embedding
- patch: Fix img for auto tune
- patch: Fix img width
- patch: Fixed a bug in prompt tuning process
- patch: Refactor text unit build at local search
- patch: Update Prompt Tuning docs
- patch: Update create_pipeline_config.py
- patch: Update prompt tune command in docs
- patch: add querying from azure blob storage
- patch: fix setting base_dir to full paths when not using file system.
- patch: fix strategy config in entity_extraction

## New Contributors
* @gukoff made their first contribution in https://github.com/microsoft/graphrag/pull/927
* @TLongP made their first contribution in https://github.com/microsoft/graphrag/pull/1039
* @fantom845 made their first contribution in https://github.com/microsoft/graphrag/pull/925
* @guangxiangdebizi made their first contribution in https://github.com/microsoft/graphrag/pull/895

**Full Changelog**: https://github.com/microsoft/graphrag/compare/v0.3.2...v0.3.3

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/graphrag/releases/tag/v0.3.3'>v0.3.3</a>.</em>

**Comment by sebastianschramm** [comment]
Is it correct to say that the incremental indexing entrypoint does not have any functionality currently because of https://github.com/microsoft/graphrag/blob/main/graphrag/index/run/run.py#L115? Or am I missing something? @dworthen

**Reply by dworthen** [reply]
I believe that is correct but including @AlonsoGuevara to confirm.

**Reply by junho85** [reply]
It looks like implementing it on incremental_indexing/main branch.
https://github.com/microsoft/graphrag/pull/1123/files#diff-a63b2d5a598d28bc57493dc8519f8a07066537626ec8bba21980757fb0f99af2R124

**Reply by AlonsoGuevara** [reply]
Yup, we have a sepparate branch for the feature as it is cunrrently under development. I had to merge some of them to main as the changes were helpful for other PRs the team was working on

**Reply by Boy124578** [reply]
That was for confirmation of to what do many more time now for update on the some time of see now

**Reply by SpikeYangRc** [reply]
When will incremental indexing be officially released?
